examples/metrics/src/main/java/io/opentelemetry/example/metrics/LongValueObserverExample.java

```diff
@@ -0,0 +1,32 @@
+package io.opentelemetry.example.metrics;
+
+import io.opentelemetry.OpenTelemetry;
+import io.opentelemetry.metrics.LongValueObserver;
+import io.opentelemetry.metrics.LongValueObserver.ResultLongValueObserver;
+import io.opentelemetry.metrics.Meter;
+
+/**
+ * Example of using {@link LongValueObserver} to measure execution time of method.
+ * Setting {@link LongValueObserver.Callback} a callback that gets executed every collection interval.
+ * Useful for expensive measurements that would be wastefully to calculate each request.
+ */
+public class LongValueObserverExample {
+
+  public static void main(String[] args) {
+    Meter sampleMeter = OpenTelemetry.getMeterProvider()
+        .get("io.opentelemetry.example.metrics", "0.5");
+    LongValueObserver observer = sampleMeter.longValueObserverBuilder("jvm_memory_usage")
+        .setDescription("should meter jvm memory usage")
+        .setUnit("byte")
+        .build();
+
+    observer.setCallback(
+        new LongValueObserver.Callback<LongValueObserver.ResultLongValueObserver>() {
+          @Override
+          public void update(ResultLongValueObserver result) {
+            result.observe(Runtime.getRuntime().totalMemory(), "jvm memory", "total");
```

Comment:
```suggestion
            result.observe(Runtime.getRuntime().totalMemory(), "kind", "total");
            result.observe(Runtime.getRuntime().freeMemory(), "kind", "free");
```

To demonstrate the advantage of using labels. Otherwise it doesn't really show why you would use them at all here.

One might prefer to report these as two separate metrics rather than metric dimensions, however.
@bogdandrutu, you're more experienced with metrics. WDYT?

Reply:
we can throw away labels from this example if it misleading